apollo-backend/src/main/java/io/logz/apollo/controllers/EnvironmentController.java

```diff
@@ -48,14 +48,15 @@ public Environment getEnvironment(int id) {
     @LoggedIn
     @POST("/environment")
     public void addEnvironment(String name, String geoRegion, String availability, String kubernetesMaster,
-                               String kubernetesToken, String kubernetesNamespace, Integer servicePortCoefficient,
+                               String kubernetesToken, String kubernetesCaCert, String kubernetesNamespace, Integer servicePortCoefficient,
```

Comment:
is `kubernetesCaCert` is the bundle of the: client.cert, client.key and ca ?

Reply:
@ofer-velich No, only the CA is required when using oauth token like we use in apollo